README.md

```diff
@@ -140,7 +140,7 @@ exit
 **Configure TechnicSolder** <br />
 
 ```bash
-chown -R www-data TechnicSolder
+chown -R www-data html
```

Comment:
Should be `TechnicSolder` to remain compatible with other instructions in the program itself and to idiot-proof it against people who might overwrite their existing html folder. Even though this requires additional webserver configuration.
```suggestion
chown -R www-data TechnicSolder
```